spec/fixtures/bolt_server/projects/bolt_server_test_project/modules/apply_helpers/tasks/apply_catalog.rb

```diff
@@ -0,0 +1,117 @@
+#! /opt/puppetlabs/puppet/bin/ruby
```

Comment:
Wondering if there's a Puppetfile for tests that we can put these external deps in. I'm guessing no?

Reply:
I thought about shelling out to r10k or even using bolt to install these as part of test setup. But i think its more work than its worth. I think for now just shipping these as duplicates is OK. If they become problematic we can update. The way i see we get the "real" test coverage in orch acceptance where we run against the actual builtin bolt content.